basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/ArtifactArchiverStep/help-excludes.html

```diff
@@ -0,0 +1,4 @@
+<div>
+Exclude artifacts matching this <a href="https://ant.apache.org/manual/dirtasks.html#patterns">Ant style pattern</a>.
```

Comment:
BTW I recommend `target="_blank"` (IIRC) for all inline help, since Jenkins configuration pages tend to crash and burn if you return to them with a Back button.